libtenzir/builtins/connectors/curl.cpp

```diff
@@ -466,20 +466,53 @@ auto parse_http_args(std::string name,
                      const operator_factory_plugin::invocation& inv,
                      session ctx) -> failure_or<connector_args> {
   auto url = std::string{};
-  auto method = std::optional<std::string>{};
+  auto body_data = std::optional<located<record>>{};
   auto params = std::optional<located<record>>{};
   auto headers = std::optional<located<record>>{};
-  argument_parser2::operator_(std::move(name))
-    .positional("url", url)
-    .named("method", method)
-    .named("params", params)
-    .named("headers", headers)
-    .parse(inv, ctx)
-    .ignore();
+  auto json = std::optional<location>{};
+  auto form = std::optional<location>{};
+  auto method = std::optional<std::string>{};
   auto args = connector_args{};
+  args.transfer_opts.default_protocol = "https";
+  auto parser = argument_parser2::operator_(name);
+  parser.positional("url", url);
+  parser.named("params", params);
+  parser.named("headers", headers);
+  parser.named("method", method);
+  if (name == "load_http") {
+    parser.named("data", body_data);
+    parser.named("json", json);
+    parser.named("form", form);
+    parser.named("chunked", args.http_opts.chunked);
+    parser.named("multipart", args.http_opts.multipart);
+  }
+  parser.named("skip_peer_verification",
+               args.transfer_opts.skip_peer_verification);
+  parser.named("skip_hostname_verification",
+               args.transfer_opts.skip_hostname_verification);
+  parser.named("verbose", args.transfer_opts.verbose);
```

Comment:
Can you rename this to `_verbose` and remove it from the docs?

Reply:
We have these on some other operators as well. I think these are useful for users to debug something. Do we still want to hide these?

Reply:
I'd say let's write a story about revisiting the verbose options of all operators. This should be done in one shot. You can leave it out for now and ship this.